openvpn-install.sh

```diff
@@ -75,11 +75,14 @@ function checkOS() {
 			fi
 		fi
 		if [[ $ID == "amzn" ]]; then
-			OS="amzn"
-			if [[ $VERSION_ID != "2" ]]; then
+			if [[ $VERSION_ID == "2" ]]; then
+				OS="amzn"
+			elif [[ "$(echo $PRETTY_NAME | cut -c 1-19)" == "Amazon Linux 2023.6" ]]; then
+				OS="amzn2023"
+			else
 				echo "⚠️ Your version of Amazon Linux is not supported."
 				echo ""
-				echo "The script only support Amazon Linux 2."
+				echo "The script only support Amazon Linux 2 or Amazon Linux 2023.6"
```

Comment:
Why 2023.6 specifically? Can't we check for 2023.*?

Reply:
> Why 2023.6 specifically? Can't we check for 2023.*?

@angristan: As @youcantgetme [pointed out](https://github.com/angristan/openvpn-install/issues/1230#issuecomment-2434456966), 2023.6 [added support for openvpn](https://github.com/amazonlinux/amazon-linux-2023/issues/79#issuecomment-2412605435) - wasn't present before that. 

The best we can do is make it compatible with 2023.6 and later versions - assuming AWS maintains openvpn support.

Reply:
Fair! Thanks